pkgs/applications/video/mpv/scripts/sponsorblock.nix

```
@@ -40,6 +40,7 @@ stdenvNoCC.mkDerivation {
   '';
 
   passthru.scriptName = "sponsorblock.lua";
+  passthru.updateScript = ./update-sponsorblock.sh;
 
   meta = with lib; {
     description = "mpv script to skip sponsored segments of YouTube videos";
```

```suggestion
    description = "Script for mpv to skip sponsored segments of YouTube videos";
```
I know you aren't touching this, but I can't let this blatant ignorance of CONTRIBUTING.md go unnoticed :)